feature.env

```diff
@@ -5,8 +5,8 @@ TOKEN_VERIFICATION_API_URL=http://localhost:9091/verification
 TOKEN_VERIFICATION_ENABLED=true
 
 EDUCATION_AND_WORK_PLAN_API_URL=http://localhost:9091
-PRISONER_SEARCH_API_URL=http://localhost:9091
-PRISON_REGISTER_API_URL=http://localhost:9091
+PRISONER_SEARCH_API_URL=http://localhost:9091/prisoner-search-api
+PRISON_REGISTER_API_URL=http://localhost:9091/prison-register-api
```

Comment:
Need to namespace these APIs so wiremock doesn't confuse common mocked endpoints.